src/CppParser/Bootstrap/Bootstrap.cs

```diff
@@ -14,11 +15,11 @@ static string GetSourceDirectory(string dir)
         {
             var directory = Directory.GetParent(Directory.GetCurrentDirectory());
 
-            while (directory != null)
+            while(directory != null)
             {
                 var path = Path.Combine(directory.FullName, dir);
 
-                if (Directory.Exists(path) &&
+                if(Directory.Exists(path) &&
```

Comment:
I am sorry for not making it clear. We do not put spaces before definitions and invocations but we do put them after operators: if, while, etc.